src/VoteDelegate.t.sol

```diff
@@ -248,6 +297,33 @@ contract VoteDelegateTest is DSTest {
         assertEq(proxy.stake(address(delegator2)), 0);
     }
 
+    function test_delegator_lock_free_after_expiration() public {
+        uint256 currMKR = gov.balanceOf(address(chief));
+
+        delegator1.approveGov(address(proxy));
+        delegator1.approveIou(address(proxy));
+
+        delegator1.doProxyLock(10_000 ether);
+        assertEq(gov.balanceOf(address(delegator1)), 0);
+        assertEq(gov.balanceOf(address(chief)), currMKR + 10_000 ether);
+        assertEq(iou.balanceOf(address(delegator1)), 10_000 ether);
+        assertEq(proxy.stake(address(delegator1)), 10_000 ether);
+
+        hevm.roll(block.number + 1);
+
+        // Warp past expiration
+        hevm.warp(block.timestamp + 9001 days);
+
+        assertTrue(block.timestamp > delegate.expiration());
```

Comment:
nit: I'd go ahead and put this assertion into all tests that test the expiration functionality